AnnoyancesFilter/Other/sections/tweaks.txt

```diff
@@ -2224,6 +2225,7 @@ thegreatdaily.com#%#Object.defineProperty(window, 'clearSelection', { get: funct
 mhwg.org,biyografi.info,seiya-saiga.com#%#//scriptlet("remove-attr", "oncontextmenu|onselectstart", "body")
 !--- HTML ----
 ! NOTE: Context menu - HTML
+ezms.link,comeinsidebox.com$$style[wildcard="user-select:*none"][max-length="500"]
```

Comment:
Why this rule is required?

Reply:
Please take a look comment of https://github.com/AdguardTeam/AdguardFilters/pull/125210/commits/28a5812783e19fa22030ab58ece91c35cd70c057.